src/prefect/runner/runner.py

```diff
@@ -485,8 +484,13 @@ async def _run_process(
         flow_run_logger.info("Opening process...")
 
         env = get_current_settings().to_environment_variables(exclude_unset=True)
-        env.update({"PREFECT__FLOW_RUN_ID": str(flow_run.id)})
-        env.update({"PREFECT__STORAGE_BASE_PATH": str(self._tmp_dir)})
+        env.update(
+            {
+                "PREFECT__FLOW_RUN_ID": str(flow_run.id),
+                "PREFECT__STORAGE_BASE_PATH": str(self._tmp_dir),
+                "PREFECT__ENABLE_CANCELLATION_AND_CRASHED_HOOKS": "false",
```

Comment:
at first glance I thought this variable name might be misleading, but it's actually a pretty clever name!